<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonForm</class>
 <widget class="QWidget" name="PersonForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>361</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>262</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>262</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_dynamic">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>2</y>
     <width>66</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>38</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="styleSheet">
    <string notr="true">  background-color: transparent; border-radius: 13px !important; </string>
   </property>
   <property name="text">
    <string>动态</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_fans">
   <property name="geometry">
    <rect>
     <x>196</x>
     <y>2</y>
     <width>66</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>38</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="styleSheet">
    <string notr="true">  background-color: transparent;
 border-radius: 13px !important; </string>
   </property>
   <property name="text">
    <string>粉丝</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_checkin">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>38</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/btn_checkin_n.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_follow">
   <property name="geometry">
    <rect>
     <x>98</x>
     <y>2</y>
     <width>66</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>38</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="styleSheet">
    <string notr="true">  background-color: transparent; 
  border-radius: 13px !important; </string>
   </property>
   <property name="text">
    <string>关注</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_dynamic_nub">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lab_follow_nub">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>0</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lab_fans_nub">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>261</width>
     <height>3</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>261</width>
     <height>3</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_vip">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-repeat: no-repeat;
text-align : left;
/*或者通过水平和垂直对齐方式来控制*/
padding-left: 20px;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>我的会员</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/information/lab_vip.png</normaloff>:/images/information/lab_vip.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="lab_vip">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>89</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>暂无会员</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/btn_right_1_h.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_grade">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-repeat: no-repeat;
text-align : left;
/*或者通过水平和垂直对齐方式来控制*/
padding-left: 20px;</string>
   </property>
   <property name="text">
    <string>等级</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/information/lab_grade.png</normaloff>:/images/information/lab_grade.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="lab_level">
   <property name="geometry">
    <rect>
     <x>187</x>
     <y>120</y>
     <width>54</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Lv.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_mark">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-repeat: no-repeat;
text-align : left;
/*或者通过水平和垂直对齐方式来控制*/
padding-left: 20px;</string>
   </property>
   <property name="text">
    <string>商城</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/information/lab_market.png</normaloff>:/images/information/lab_market.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_mesg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-repeat: no-repeat;
text-align : left;
/*或者通过水平和垂直对齐方式来控制*/
padding-left: 20px;</string>
   </property>
   <property name="text">
    <string>个人信息设置</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/information/lab_mesg.png</normaloff>:/images/information/lab_mesg.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_bing">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-repeat: no-repeat;
text-align : left;
/*或者通过水平和垂直对齐方式来控制*/
padding-left: 20px;</string>
   </property>
   <property name="text">
    <string>绑定社交账号</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/information/lab_bing.png</normaloff>:/images/information/lab_bing.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_server">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>280</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-repeat: no-repeat;
text-align : left;
/*或者通过水平和垂直对齐方式来控制*/
padding-left: 20px;</string>
   </property>
   <property name="text">
    <string>我的客服</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/information/lab_service.png</normaloff>:/images/information/lab_service.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_out">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-repeat: no-repeat;
text-align : left;
/*或者通过水平和垂直对齐方式来控制*/
padding-left: 20px;</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/information/lab_out.png</normaloff>:/images/information/lab_out.png</iconset>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>261</width>
     <height>3</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>261</width>
     <height>3</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/btn_right_1_h.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/btn_right_1_h.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/btn_right_1_h.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>240</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/btn_right_1_h.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>290</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/btn_right_1_h.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>340</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/btn_right_1_h.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>lab_vip</zorder>
  <zorder>lab_fans_nub</zorder>
  <zorder>lab_follow_nub</zorder>
  <zorder>lab_dynamic_nub</zorder>
  <zorder>btn_dynamic</zorder>
  <zorder>btn_fans</zorder>
  <zorder>btn_checkin</zorder>
  <zorder>btn_follow</zorder>
  <zorder>line</zorder>
  <zorder>line_2</zorder>
  <zorder>btn_vip</zorder>
  <zorder>btn_grade</zorder>
  <zorder>lab_level</zorder>
  <zorder>btn_mark</zorder>
  <zorder>btn_mesg</zorder>
  <zorder>btn_bing</zorder>
  <zorder>btn_server</zorder>
  <zorder>btn_out</zorder>
  <zorder>line_3</zorder>
  <zorder>line_4</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>label_10</zorder>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
